main/saturn-node.js

```diff
@@ -30,12 +28,6 @@ let webUrl
 /** @type {string | undefined} */
 let apiUrl
 
-const ConfigKeys = {
-  FilAddress: 'saturn.filAddress'
```

Comment:
Here we use the key `saturn.filAddress`, but the new code uses the key `station.filAddress`. I think that will break compatibility.

Maybe we can use electron-store migrations to "upgrade" the config from the key name to the new key?
https://www.npmjs.com/package/electron-store#migrations

Alternatively, we can implement a fallback in `getFilAddress` to check the old key if there is no value stored under the new key.

What do you think?

Considering how few users we have right now, I can live with breaking the compatibility too. Having said that, I think this is an excellent opportunity to learn how to handle breaking changes in the future when they become more of a problem.

Reply:
Great idea! I've implemented migrations, and also made some other keys more consistent